index.html

```diff
@@ -235,8 +235,10 @@ <h1 class="ui-text-title">Flow Launcher</h1>
       </section>
       <section class="ui-section-customer">
         <div class="ui-layout-container">
+          <p style="font-size:14px; color:#898989; text-align: center; margin-bottom:1rem">Sponsored by</p>
```

Comment:
Would it be better to add this style as a new CSS class?

Reply:
Applied.